solr/licenses/org.osgi.annotation.bundle-LICENSE-ASL.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
If the LICENSE and NOTICE files are the same for these two osgi artifacts, it is sufficient with one named `org.osgi.annotation-LICENSE-ASL.txt`

Reply:
All artifacts that start with `org.osgi` come from the same repo and have the same `NOTICE` file. The current `validateJarLicenses` Gradle task only splits artifact names on `-`, not on `.`, but I can improve it in a separate PR.